bin/lock

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bin/ruby
```

Comment:
I'm open to suggestions but for now this is executed manually. What do you suggest specifically?